dashboards/scanned-events-query-governance/query-governance.json

```diff
@@ -0,0 +1,1397 @@
+{
+  "name": "Query Governance",
+  "description": null,
+
+  "pages": [
+    {
+      "name": "BSE Governance",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 3,
+            "height": 9
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![New Relic](https://newrelic.com/sites/default/files/2022-05/MediaAsset_Tiles_RGB_POS_HZ.svg) \n# Governance Dashboard ![New Relic logo](https://newrelic.com/static-assets/images/icons/avatar-newrelic.png)\n## Get insights on how your data is access at deep granularity.\n> Base table example:\n>>`FROM NrdbQuery SELECT sum(scannedEvents) / 1e9 AS 'Billion SE' SINCE 1 week ago until now`\n## Accounts in this dashboard:\nEnable crossaccount dashboards to be able to load several accounts at once\n| **Account Name**                               | **Account ID** |\n| ---------------------------------------------- | -------------- |\n| Account Name 1  | ######        |\n| Account Name 1                                        | ######         |\n| Account Name 1                                  | ######        |\n| Account Name 1                   | ######        |\n| Account Name 1                       | ######        |\n\n## Scanned Events:\n- Total Volume\n- Daily Trends\n## Unique User Trends:\n- Monthly Volume of users scanning events\n- User Trend for scanned events\n### Monthly volume by source:\n#### Where is are the highest percentages of scanned events coming from?\n##### Main sources include:\n- Dashboards\n- Nerdgraph\n- Distributed Tracing\n- Legacy API\n## Resources:\n[How to query your scanned events](<https://docs-preview.newrelic.com/docs/query-based-pricing?utm_source=customer-dashboard&utm_medium=asset&utm_campaign=quickstart&utm_content=custom-dashboard>)\n"
```

Comment:
```suggestion
            "text": "![New Relic](https://newrelic.com/sites/default/files/2022-05/MediaAsset_Tiles_RGB_POS_HZ.svg) \n# Governance Dashboard ![New Relic logo](https://newrelic.com/static-assets/images/icons/avatar-newrelic.png)\n## Get insights on how your data is accessed at deep granularity.\n> Base table example:\n>>`FROM NrdbQuery SELECT sum(scannedEvents) / 1e9 AS 'Billion SE' SINCE 1 week ago until now`\n## Accounts in this dashboard:\nEnable crossaccount dashboards to be able to load several accounts at once\n| **Account Name**                               | **Account ID** |\n| ---------------------------------------------- | -------------- |\n| Account Name 1  | ######        |\n| Account Name 1                                        | ######         |\n| Account Name 1                                  | ######        |\n| Account Name 1                   | ######        |\n| Account Name 1                       | ######        |\n\n## Scanned Events:\n- Total Volume\n- Daily Trends\n## Unique User Trends:\n- Monthly Volume of users scanning events\n- User Trend for scanned events\n### Monthly volume by source:\n#### Where is are the highest percentages of scanned events coming from?\n##### Main sources include:\n- Dashboards\n- Nerdgraph\n- Distributed Tracing\n- Legacy API\n## Resources:\n[How to query your scanned events](<https://docs-preview.newrelic.com/docs/query-based-pricing?utm_source=customer-dashboard&utm_medium=asset&utm_campaign=quickstart&utm_content=custom-dashboard>)\n"
```